doc/sphinxext/allow_nan.py

```diff
@@ -0,0 +1,59 @@
+from sklearn.utils import all_estimators
```

Comment:
To be consistent with the class name, can you rename this: `doc/sphinxext/allow_nan_estimators.py`?

Reply:
Updated :)